ansible/roles/anitya-dev/tasks/db.yml

```diff
@@ -43,8 +43,14 @@
       creates: /home/vagrant/.db-imported
 
 - name: Create /home/vagrant/alembic.ini
+  become_user: "{{ ansible_env.SUDO_USER }}"
   copy: src=/home/vagrant/devel/alembic.ini dest=/home/vagrant/alembic.ini remote_src=True
 
+- name: Create /home/vagrant/alembic.ini
+  replace:
+    dest: /home/vagrant/alembic.ini
+    regexp: "^script_location = alembic*$"
+    replace: "script_location = devel/alembic"
```

Comment:
Ah, it becomes the user who's using sudo, so in the Vagrant VM's case that's the ``vagrant`` user.